src/main/java/com/uwaterloo/connect/repository/AttachmentRepository.java

```diff
@@ -0,0 +1,20 @@
+package com.uwaterloo.connect.repository;
+
+import com.uwaterloo.connect.model.Attachment;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+
+@Repository
+public interface AttachmentRepository extends JpaRepository<Attachment, Integer> {
+
+    @Query("select a from Attachment a where a.postId= :postId")
```

Comment:
You can get rid of the query and replace your function name with findByPostId(). Spring will take care of the rest

Reply:
Fixed in local